lib/runtime/transaction_validity_test.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package runtime
+
+import (
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/transaction"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_UnmarshalTransactionValidity(t *testing.T) {
```

Comment:
Can you cover the code paths for errors as well? If you can't make it, then change the `return` to panics. That's also a good way to find if an error should be an error or a panic (progrming error)

Reply:
Yeah I do not believe I can, can update